spec/support/idv_examples/usps_address_confirmation.rb

```diff
@@ -0,0 +1,52 @@
+shared_examples 'idv usps address confirmation' do |sp|
```

Comment:
I'd like to rename this. Having `usps_address_confirmation.rb` next to `usps_verification.rb` is a little bit confusing.